README.md

```diff
@@ -24,7 +24,10 @@ source ./venv/bin/activate
 pip3 install -r requirements.txt 
 ```
 
-- Please download Firebase Admin SDK service json file from Firebase Console and put that in project's root directory. Rename file to firebase_cred.json
+- Please download Firebase Admin SDK service json file from Firebase Console, You can download the file by following below steps:
+    -  Go to `Project Settings/Service accounts`
```

Comment:
@raghavjoshi789 This list should be a numbered list as the steps follow an order.